### PR TITLE
add more permissions for irsa operator

### PR DESCRIPTION
**Pull request by whites11**
we use even more actions to ensure OIDC provider objects are correct

### CODE REVIEW

File: capa-controller-role/irsa-operator-policy.json [hunk 1]
```diff
@@ -11,6 +11,9 @@
                 "iam:UntagOpenIDConnectProvider",
                 "iam:ListOpenIDConnectProviders",
                 "iam:GetOpenIDConnectProvider",
+                "iam:UpdateOpenIDConnectProviderThumbprint",
+		        "iam:RemoveClientIDFromOpenIDConnectProvider",
+		        "iam:AddClientIDToOpenIDConnectProvider",
```
Comment (calvix) [inline capa-controller-role/irsa-operator-policy.json:16]: wrong ident

Comment (whites11) [reply capa-controller-role/irsa-operator-policy.json]: yeah, fixed

File: aws-operator-role/iam-policy.json [hunk 2]
```diff
@@ -72,9 +75,7 @@
                 "s3:GetObject",
                 "s3:ListBucket",
                 "s3:PutBucketAcl",
-                "s3:PutBucketEncryption",
```
[the remark this answers is not in the document]
Comment (whites11) [reply aws-operator-role/iam-policy.json:75]: They don't exist.

**Review by njuettner — APPROVED** [review]

File: aws-operator-role/iam-policy.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (njuettner) [reply aws-operator-role/iam-policy.json:75]: Thanks <3

**Pull request merged**